src/domain/DemoData.ts

```diff
@@ -18,21 +18,27 @@ export default class DemoData {
         this.knightLogic = knightLogic
     }
 
-    async store() {
+    async create() {
         log.admin('Creating demo data...')
 
         let tx = new PgTransaction(this.pool)
 
-        let lAdress = new Adress('Gardenstreet', '42c', '12345', 'Gardencity', 'Gardencountry')
+        let lAddress = new Address('Gardenstreet', '42c', '12345', 'Gardencity', 'Gardencountry')
 
         let luisa = new Knight({
             name: 'Luisa',
-            adress: lAdress
+            address: lAddress
         })
 
-        log.admin('luisa', luisa)
-        let result = await this.knightLogic.store(luisa, tx)
-        log.admin('created knight', result)
+        //await this.knightLogic.store(luisa, tx)
 
+        let alreadyExist = await this.knightLogic.read({name: 'Luisa'}, tx)
```

Comment:
Wie wäre es denn hier mit einer Funktion, in die ich eine beliebige Instanz reingeben kann. Die Funktion überprüft dann erstmal, ob die Instanz schon existiert und ruft dann bei Bedarf store auf.